proving_system/stark/benches/criterion_stark.rs

```diff
@@ -7,47 +17,73 @@ pub fn proof_benchmarks(c: &mut Criterion) {
     let mut group = c.benchmark_group("STARK");
     group.sample_size(10);
 
-    for trace_length in [8, 128, 256, 512, 1024].iter() {
-        group.throughput(Throughput::Bytes(*trace_length as u64));
-        group.bench_with_input(
-            BenchmarkId::new("Simple Fibonacci", trace_length),
-            trace_length,
-            |b, &t| {
-                b.iter(|| functions::stark::prove_fib(t));
-            },
-        );
+    for &trace_length in [64, 128, 256, 512].iter() {
+        let (trace, fibonacci_air) = generate_fib_proof_params(trace_length);
+        group.throughput(Throughput::Elements(trace_length as u64));
+        group.bench_function(BenchmarkId::new("Simple Fibonacci", trace_length), |b| {
+            b.iter(|| black_box(prove(black_box(&trace), black_box(&fibonacci_air))));
+        });
     }
 
-    group.bench_function("2 column Fibonacci", |bench| {
-        bench.iter(functions::stark::prove_fib_2_cols);
-    });
-
-    group.bench_function("Fibonacci F17", |bench| {
-        bench.iter(functions::stark::prove_fib17);
-    });
+    for &trace_length in [64, 128, 256, 512].iter() {
+        let (trace, fibonacci_air) = generate_fib_2_cols_proof_params(trace_length);
+        group.throughput(Throughput::Elements(trace_length as u64));
+        group.bench_function(BenchmarkId::new("2 column Fibonacci", trace_length), |b| {
+            b.iter(|| black_box(prove(black_box(&trace), black_box(&fibonacci_air))));
+        });
+    }
 
-    group.bench_function("Quadratic AIR", |bench| {
-        bench.iter(functions::stark::prove_quadratic);
-    });
+    // TODO: this benchmarks panic
```

Comment:
Do we know why?

Reply:
I checked the integration tests and updated the parameters used in the benches to match those. Now they (hopefully) don't panic!